trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_RHS_matrix.py

```diff
@@ -57,13 +57,17 @@ def compute_RHS_matrix(n_sw,n_cw,delta,phi,conditions,geometry,propeller_wake_mo
         #-------------------------------------------------------------------------------------------------------
         # PROPELLER SLIPSTREAM MODEL
         #-------------------------------------------------------------------------------------------------------         
-        if propeller_wake_model and ('propeller' in propulsor.keys()):   
+        if propeller_wake_model and (('propeller' in propulsor.keys()) or ('rotor' in propulsor.keys())):   
```

Comment:
what happens when both the rotor and prop are spinning like in a transition?

Reply:
The only time any wake analysis will be done is if the model boolean is flagged on, regardless of if the vehicle is in transition or not. It can handle the transition though